.nvmrc

```diff
@@ -1 +1 @@
-10
\ No newline at end of file
+14
```

Comment:
What does this change do?

Reply:
Oh I wanted to change the default Node version (for `nvm`) to be Node 14, matching what the `publish` workflow uses.